samples/system-test/spanner.test.js

```diff
@@ -134,7 +134,7 @@ describe('Spanner', () => {
         },
         gaxOptions: GAX_OPTIONS,
       });
-      await operation.promise();
+      return operation.promise();
```

Comment:
I don't understand why we'd need to switch to returning a promise here, rather than the `await`, I believe it should wait for async operations to finish if they're awaited?

Reply:
`before` is able to accept a `promise` and will wait for it to finish. In case of reject it will not proceed further.

Reply:
It does seem that way according to https://mochajs.org/#working-with-promises.

Although the section below that (https://mochajs.org/#using-async-await) says that `await` should also work. I don't know why it didn't. The tests clearly failed because the instance wasn't created.